### PR TITLE
GEODE-7428: change log level in management API

### PR DESCRIPTION
**Pull request by jmelchio**
Simple change to log level in management API to avoid logs being too chatty

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [x] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [x] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jmelchio** [comment]
@jinmeiliao @onichols-pivotal @dschneider-pivotal @dhemery

File: geode-web-management/src/main/java/org/apache/geode/management/internal/rest/ManagementLoggingFilter.java [hunk 1]
```diff
@@ -31,21 +31,12 @@
...
 
   @Override
   protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response,
       FilterChain filterChain) throws ServletException, IOException {
 
-    if (!ENABLE_REQUEST_LOGGING) {
-      filterChain.doFilter(request, response);
-      return;
-    }
-
```
Comment (onichols-pivotal) [inline geode-web-management/src/main/java/org/apache/geode/management/internal/rest/ManagementLoggingFilter.java]: This change seems to have an adverse performance impact.  Even if debug logging is not enabled, we will now always wrap and capture the request and response.  Before, the wrapping and capturing was able to be skipped.  Why not keep the same logic as before, just change the test to from system-property check to isDebugEnabled check?

Comment (jmelchio) [reply geode-web-management/src/main/java/org/apache/geode/management/internal/rest/ManagementLoggingFilter.java]: Yes, that's a good point, I'll make that change.

**Comment by onichols-pivotal** [comment]
looks like ManagementRequestLoggingDistributedTest needs to be updated

**Pull request merged**